src/test/scala/nl.knaw.dans.easy.validatebag/EasyValidateDansBagServletSpec.scala

```diff
@@ -35,16 +37,14 @@ class EasyValidateDansBagServletSpec extends TestSupportFixture
     if (isAvailable(triedAgreementSchema, triedFileSchema, triedDdmSchema)) {
       // offline schema's -> no app -> no servlet
       // (otherwise test suite aborts, now tests are ignored)
-      try {
-        val configuration = Configuration(
-          testVersion,
-          createProperties(),
-          Seq(new URI("http://creativecommons.org/licenses/by-sa/4.0")),
-        )
-        val app = new EasyValidateDansBagApp(configuration)
-        val servlet = new EasyValidateDansBagServlet(app)
-        addServlet(servlet, "/*")
-      }
+      val configuration = Configuration(
+        testVersion,
+        createProperties(),
```

Comment:
When replacing this with null, the error is not swallowed. So the try clause was not needed.